.github/workflows/test.yml

```diff
@@ -245,9 +245,10 @@ jobs:
     steps:
       - run: echo ${{ matrix.projectName }}
       - uses: actions/checkout@v3
+      - uses: ./.github/actions/checkout-submodules
         with:
-          submodules: true
-          token: ${{ secrets.SUBMODULES_TOKEN }}
+          submodule_token: ${{ secrets.SUBMODULES_TOKEN }}
+          submodule_branch: "next"
```

Comment:
Should we try using the current branch here aswell?

Reply:
For the test i am not sure, because the name should be the exact name as in the enterprise repo which could be hard to maintain, and in addition It will break in the following case, if we have one PR in main repo and multiply branches in the enterprise repo (due the possible complexity)